### PR TITLE
Created Button and card

### PR DESCRIPTION
**Pull request by aryanandarnav13**

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
Good to merge

**Comment by yash9263** [comment]
Looks good to me to merge

**Review by yash9263 — APPROVED** [review]

File: css/card.css [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      border-top: 1px solid #D7DBDD;
+      background-color: #EEF3F5;
+    }
+    .card-tag2 {
+        text-align: center;
+        color: #57727C;
+        font-size: 1rem;
+        font-weight: 700;
+        padding: 0.5rem 0.5rem 0rem ;
+
```
Comment (yash9263) [inline css/card.css:46]: extra line

File: css/card.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    bottom: 0;
+    height: 1.3rem;
+    position: absolute;
+    right: 5%;
+    
+    width: 1.3rem;
+  }
+
+  .badge-status1{
+    background-color: rgb(123, 228, 102);
```
Comment (yash9263) [inline css/card.css:71]: can use variables for color

File: css/card.css [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  .closebtn:hover {
+    color: black;
+  }
+
+
+body{
+    box-sizing: border-box;
+}
+
+  a{
```
Comment (yash9263) [inline css/card.css:93]: use consistent indentation

File: css/card.css [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+  .background{
+      grid-column: 1/2;
+      grid-row: 1/2;
+  }
+
+
+
+
+
+
```
Comment (yash9263) [inline css/card.css:130]: Extra space

File: css/card.css (hunk 1 above)
Comment (aryanandarnav13) [reply css/card.css:46]: Yes, I'll correct

File: css/card.css (hunk 3 above)
Comment (aryanandarnav13) [reply css/card.css:93]: Thankyou for reviewing

**Pull request merged**